packages/grafana-data/src/events/EventBusGroup.ts

```diff
@@ -0,0 +1,67 @@
+// import { Unsubscribable, Subscription, Observable } from 'rxjs';
+// import { BusEvent, BusEventType, BusEventHandler, EventBus } from './types';
+
+// /**
+//  * Handles unsubscribing to all events subscribed through this group
+//  */
+// export class EventBusGroup implements EventBus {
```

Comment:
hmm, what was the idea behind this one? Make it easier to unsubscribe to multiple subscriptions?

Reply:
yea, exactly. But think I will solve this some other way